android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Well, that's one opinion. We are definitely ready to consider it, but the problem is, the more code we duplicate, harder it becomes to maintain it. Changing one model breaks the other 3

There should be a better solution to that

@the-dagger @mananwason Please guide